### PR TITLE
Refactor: Delete `GAButton` and use styled components directly.

### PR DESCRIPTION
**Pull request by eddsaura**
## Refactor: Delete GAButton and use styled components directly.

Delete GAButton which added complexity, and use the styled components directly with onClick function to use Google Analytics.

### CODE REVIEW

**Comment by stooa-dev** [comment]
recheck

File: frontend/components/App/Participants/Participant.tsx [hunk 1]
```diff
@@ -71,18 +73,21 @@ const ParticipantComponent: React.FC<{ participant: Participant; speaker?: boole
...
-              className="icon"
-              variant="link"
-              event={{
-                action: 'Linkedin',
-                category: 'Participants',
-                label: window.location.href
-              }}>
+              className="icon">
+              {' '}
+              index
```
Comment (italodr) [inline frontend/components/App/Participants/Participant.tsx:88]: Are those lines necessary?

File: frontend/components/App/Participants/Participant.tsx [hunk 2]
```diff
@@ -51,18 +50,21 @@ const ParticipantComponent: React.FC<{ participant: Participant; speaker?: boole
...
               rel="noreferrer"
-              className="icon"
-              variant="link"
-              event={{
-                action: 'Twitter',
-                category: 'Participants',
-                label: window.location.href
-              }}>
+              className="icon">
+              {' '}
```
Comment (italodr) [inline frontend/components/App/Participants/Participant.tsx:65]: I think this is not necessary

File: frontend/components/App/Participants/Participant.tsx (hunk 1 above)
Comment (eddsaura) [reply frontend/components/App/Participants/Participant.tsx:88]: No, you are right.

File: frontend/components/App/Participants/Participant.tsx (hunk 2 above)
Comment (eddsaura) [reply frontend/components/App/Participants/Participant.tsx:65]: Actually this was Prettier but when I delete it, it wont appear anymore, strange.

File: frontend/pages/fishbowl/thankyou/[fid].tsx [hunk 3]
```diff
@@ -14,7 +14,7 @@ import dynamic from 'next/dynamic';
 import useTranslation from 'next-translate/useTranslation';
 
 import { ROUTE_FISHBOWL_CREATE, ROUTE_NOT_FOUND } from 'app.config';
-import { dataLayerPush } from 'lib/analytics';
+import { dataLayerPush, pushEventDataLayer } from 'lib/analytics';
```
Comment (italodr) [inline frontend/pages/fishbowl/thankyou/[fid].tsx:17]: I would consider renaming `dataLayerPush` function. In my opinion it is confusing as they both push events to Analytics

Comment (eddsaura) [reply frontend/pages/fishbowl/thankyou/[fid].tsx:17]: Yeah, maybe we should use something like `pageView...`. Should we fix this in the next PR?

File: frontend/pages/index.tsx [hunk 4]
```diff
@@ -30,6 +29,8 @@ import BillboardMobAnimPath from 'ui/animations/home/billboard-mobile.json';
 import MorphBillAnimPath from 'ui/animations/home/billboard-morph.json';
 import Morph2BillAnimPath from 'ui/animations/home/billboard-morph-2.json';
 import KeyBenefit2MorphPath from 'ui/animations/home/keybenefit2-morph.json';
+import Button, { ButtonStyledLink } from 'ui/Button';
```
Comment (italodr) [inline frontend/pages/index.tsx:32]: Fix this warning please

**Review by italodr — APPROVED** [review]

**Pull request merged**